core/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/diskModels.scala

```diff
@@ -52,4 +80,9 @@ object DiskType extends Enum[DiskType] {
   final case object SSD extends DiskType {
     override def googleString: String = "pd-ssd"
   }
+  def stringToDiskType(string: String): DiskType =
```

Comment:
can you use `sealerate.values.map(obj => obj.googleString -> obj` for `stringToDiskType`

Reply:
Decided to revert back to Enum and use the given `withNameOption` method instead of writing our own here